__device-tests__/gutenberg-editor-gallery-visual.test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+const { blockNames } = editorPage;
+import { takeScreenshot } from './utils';
+
+describe( 'Gutenberg Editor Visual test for Gallery Block', () => {
+	it( 'should be able to render the placeholder correctly', async () => {
+		await editorPage.addNewBlock( blockNames.gallery );
+		await editorPage.closePicker();
+
+		// Visual test check
+		const screenshot = await takeScreenshot();
+		expect( screenshot ).toMatchImageSnapshot();
+
+		await editorPage.removeBlockAtPosition( blockNames.gallery );
+	} );
+
+	it( 'should be able to render a gallery correctly', async () => {
+		await editorPage.setHtmlContent(
+			[ e2eTestData.galleryBlock ].join( '\n\n' )
+		);
+		const galleryBlock = await editorPage.getBlockAtPosition(
+			blockNames.gallery
+		);
+		expect( galleryBlock ).toBeTruthy();
+
+		// Wait for images to load
+		await editorPage.driver.sleep( 5000 );
+
+		// Visual test check
+		const screenshot = await takeScreenshot();
+		expect( screenshot ).toMatchImageSnapshot();
```

Comment:
I noticed the previous test has what appears to be clean up code removing the block. Should all tests including this one have that as well? This question applies to `__device-tests__/gutenberg-editor-group-visual.test.js` as well. 
```suggestion
		expect( screenshot ).toMatchImageSnapshot();
		
		await editorPage.removeBlockAtPosition( blockNames.gallery );
```

Reply:
It will depend on the test, for this case, to start with a "blank" canvas in the second test you need to remove the blocks you created before, as the other tests in the file will run in the same session.